[CIAPP-6240] Clarifying documentation for CI Visibility Test Summaries (#17248)
DataDog/documentation: content/en/continuous_integration/guides/pull_request_comments.md

<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
{{< /site-region >}}

If you're using the Test Visibility product, Datadog can integrate with GitHub to show test results summaries directly
in your pull requests. The summary contains an overview of the tests executions, flakiness information and
error messages for failed tests.
With this report, the developers get instant feedback about the results of their tests, and they are able to
debug any failed/flaky tests without leaving the pull requests view.

{{< img src="ci/github_comments_light.png" alt="Datadog GitHub pull request comment preview" style="width:100%;">}}

### Compatibility

This integration is only available for test services hosted on `github.com`.

> ## Enable test summaries

Reviewer: ```suggestion
## Enable test summaries in GitHub
```

> Enabling test summaries is very quick and can be done in two simple steps. The first step is to install the [GitHub integration][1]:

Reviewer: ```suggestion
You can enable test summaries in GitHub by following the next steps:

1. Install the [GitHub integration][1]:
```

Author: This is how I wrote that in principle, but then you have nested ordered lists. Something like:

1. Install Github integration:
    a. Navigate to the Configuration tab...
    b. Give the application...
    
2. Enable test summaries for a test service...

Which is something that I didn't really see in our UI in other places. Do you think that would be better?

Reviewer: Yes, I'd say so IMO. I could locate an example for Jenkins (unordered though) https://github.com/DataDog/documentation/blob/master/content/en/continuous_integration/pipelines/jenkins.md?plain=1#L794-L805

Author: 👍 Changed to be an ordered list!

1. Navigate to the **Configuration** tab on the [GitHub integration tile][2] and click **+ Create GitHub App**.
2. Give the application read and write permissions for pull requests.

> The second step is to enable test summaries for a test service from the Test Service Settings page or from the commit/branch page.

Reviewer: ```suggestion
2. Enable test summaries for a test service. You can do that either using the Test Service Settings page or from the commit/branch page.
```

<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
{{< /site-region >}}

If you're using the Test Visibility product, Datadog can integrate with GitHub to show test results summaries directly
in your pull requests. The summary contains an overview of the tests executions, flakiness information and
error messages for failed tests.

> With this report, developers get instant feedback about the results of their tests, and they are able to

Reviewer 2: ```suggestion
With this report, developers get instant feedback about their tests results, including the ability to
```

debug any failed/flaky tests without leaving the pull request view.

{{< img src="ci/github_comments_light.png" alt="Datadog GitHub pull request comment preview" style="width:100%;">}}

### Compatibility

This integration is only available for test services hosted on `github.com`.

## Enable test summaries

You can enable test summaries in pull requests with the following steps:

1. Install the [GitHub integration][1]:
   1. Navigate to the **Configuration** tab on the [GitHub integration tile][2] and click **+ Create GitHub App**.
   2. Give the application read and write permissions for pull requests.

> 2. Enable test summaries for one or more test services. It can be done from the Test Service Settings page or from the commit/branch page.

Author: Changed!
